internal/distro/rhel90/pipelines.go

```diff
@@ -1199,7 +1199,7 @@ func simplifiedInstallerTreePipeline(repos []rpmmd.RepoConfig, packages []rpmmd.
 	p.AddStage(osbuild.NewBuildstampStage(buildStampStageOptions(arch, product, osVersion, variant)))
 	p.AddStage(osbuild.NewLocaleStage(&osbuild.LocaleStageOptions{Language: "en_US.UTF-8"}))
 	p.AddStage(osbuild.NewSystemdStage(systemdStageOptions([]string{"coreos-installer"}, nil, nil, "")))
```

Comment:
drop this line as well (from https://github.com/osbuild/osbuild-composer/pull/1752/files#diff-96e355e931e7a80acc491324230af1bd2af9cccb72609a76c246d923e02a773eL1202)

Reply:
Good catch! :+1: